cartographer/mapping_3d/local_trajectory_builder.h

```diff
@@ -40,10 +40,24 @@ namespace mapping_3d {
 // without loop closure.
 class LocalTrajectoryBuilder {
  public:
+  // The following two structs are used for representing the result of adding
+  // range data in local SLAM (using 'LocalTrajectoryBuilder::AddRangeData()').
+  // 'nullptr' is returned for 'MatchingResult' in case if the range data
+  // accumulation is not yet complete. Otherwise, a pointer is returned to a
+  // 'MatchingResult' instance which contains the accumulated 'RangeData',
```

Comment:
This is redundant and already better conveyed by the code. If you want to add more information about a specific part of the data: Add a comment there.